app/register/register.go

```diff
@@ -583,7 +585,7 @@ func (c clusterRegisterUtil) GetClusters(astraHost, cloudId, apiToken string) (G
 	}
 
 	if clustersResp.StatusCode != http.StatusOK {
-		return clustersRespJson, fmt.Sprintf("GET call to %v failed with http status code %v", url, clustersResp.StatusCode), errors.New("get clusters failed " + strconv.Itoa(clustersResp.StatusCode))
+		return clustersRespJson, fmt.Sprintf("GET call to %v failed with http status %v", url, clustersResp.Status), errors.New("get clusters failed " + strconv.Itoa(clustersResp.StatusCode))
```

Comment:
is response.Body not available in all cases? It looks like it is missing from some of these but not sure if that's expected

Reply:
I just pushed a change rewriting a bunch of these so they're standardized, so that might clear up your question a bit, but the response body has to be parsed before we can return it. For some reason this check is above the attempt to parse the body though. I'll re-arrange so we have that available